### ISSUE TITLE
tool for dessesambling the pidgen.dll?

### ISSUE THREAD

**Issue by arminmoradi96** [UMSKT/UMSKT#146]
Hello,

@Neo-Desktop has recommendet me to dessamembly the pidgen.dll gidra. I am blind, gidra is not accessible with screenreader software. Do you have an idear, how i can read out the pidgen.dll and look in the setupp.ini for key generation?

Regards
Armin

**Comment by sebada11** [comment]
You can check here: https://stackoverflow.com/questions/5197397/how-do-i-decompile-a-dll-file
